code/modules/clothing/head/misc.dm

```diff
@@ -280,5 +280,5 @@
 
 /obj/item/clothing/head/crown/fancy
 	name = "magnificent crown"
-	desc = "A crown worn by only the highest emperors of the land."
+	desc = "A crown worn by only the highest emperors of the <s>land</s> space."

```

Comment:
I am still figuring out how to change this to be more correct but at the same time not removing the joke